template/6-non_functional_requirements.md

```diff
@@ -2,22 +2,46 @@
-- Do not store the location of users online
+User authentication is managed by the third-party Google Authentication system. Consequently, credential data does not pass through our backend, ensuring that none of this highly sensitive information (such as passwords and tokens) is retained or exposed to risk on our side. This approach leverages Google's robust security infrastructure, providing a secure and reliable authentication process for our users.
 
-*Which privacy features do you need from the phone?*
+### Data Retention Policy
+
+Proxima collects only the necessary data required for the proper functioning of the application. This includes the user's representation in the application (such as username, display name, and join date) as well as the content published by the user.
+
+An important consideration is the deletion of accounts as required by law. Our policy in this scenario is to delete all content linked to the user account. Specifically, this includes the user's personal data as well as any content they have published, such as posts and comments. This ensures compliance with legal requirements and respects user privacy.
+
+### Location Policy
+
+The primary privacy consideration for Proxima concerns the management of user location data. Recognizing the sensitivity of this information, we have established a strict policy regarding its usage.
+
+Proxima will not store any user location data on the backend. The only location data stored in the backend are the locations of posts. While these locations are initially linked to a user at the time of posting, they quickly become non-relevant as users move to different places. This approach ensures that users' location privacy is maintained while still enabling the core functionalities of the application.
+
+### Moderation
+
+Moderation is a crucial aspect of any public social media platform, and our application is no exception. Our policy regarding moderation will be based on a reporting system complemented by AI analysis and human judgment.
+
+If content is reported by multiple users, it will first be evaluated by an AI system to determine its appropriateness. If the AI model cannot fully decide on the nature of the content, human moderators will be engaged to make the final decision on whether to remove the content or not. This combined approach ensures that moderation is efficient, accurate, and fair, maintaining a safe and respectful environment for all users.
 
-- Privacy
-- Moderation
-- Authentication
 
 ## Adoptions, Scalability and Availability
 
-*What kind of traffic patterns do you expect to see?*
+### Adoption
+
+1. **Quick Engagement**: Upon the initial launch of the app, users should be able to view and post content with minimal steps to reduce conversion costs and facilitate quick engagement.
+2. **Challenge Tour**: When accessing the challenges page for the first time, users should receive a brief tour explaining how the challenges work and the logic behind Centauri points.
```

Comment:
I have a strong struggle to understand the difference between functional and non-functional requirements (as you can guess reading my other comments). Isn't this a functional one?

Moreover, reading the `Adoption` part of Pocket Campus' PRD, they speak of adoption metrics here. Should we do the same? This might be duplicated with `Success criteria` however. What do you think?

Reply:
That's a good point, thanks !
I think that there definition of functional/non-functional requirements is pretty vague personnaly.
For instance, in the PRD of Pocket campus, they have 
"The application must, by default, have all push notifications disabled to
avoid unnecessary digital distractions for the users."
This could be seen as a functional requirement in my opinion.

I have tried my best to keep those requirements separated while treating the relevant points. Thus I belive those are ok in this part.

I personnaly think that talking about Adoption metrics doesn't add a lot given that it is well detailled in the `Sucess criteria` part as you said.